mail/mailer.go

```diff
@@ -121,11 +121,14 @@ func New(server, port, username, password string, from mail.Address, stats stats
 // New constructs a Mailer suitable for doing a dry run. It simply logs each
 // command that would have been run, at debug level.
 func NewDryRun(from mail.Address, logger blog.Logger) *MailerImpl {
+	statter, _ := statsd.NewNoopClient(nil)
+	stats := metrics.NewStatsdScope(statter)
```

Comment:
Hm. It's not a drop in replacement:
`cannot use stats (type metrics.Scope) as type *metrics.StatsdScope in field value: need type assertion `
I'll have to rejig tomorrow AM. :wave: